### DISCUSSION TITLE
How do i remove links from apps ?

### DISCUSSION THREAD

**Discussion by xd003** [mue/mue#746] [General]
i have added certain links in Navbar -> Apps. I want to remove some of them but i can't find an option to remove them. Can anyone please help me out on the same

**Comment by davidcralph** [comment]
This appears to be a bug. We will fix the apps feature in the 7.1.1 update which will release as soon as possible.

**Reply by xd003** [reply]
Should i open a github issue for this ? i am guessing its not needed as there aren't too many there so this wouldn't get lost in the mix

**Reply by davidcralph** [reply]
We've just fixed this 🙂 I don't think it's necessary to open an issue now, but if you feel it'd be helpful for people to know you can (it'll get closed instantly)
